src/ovp_pipeline/commands/task_dispatch.py

```
@@ -0,0 +1,599 @@
+
+
+class NoHandlerError(DispatchError):
+    pass
+
+
+def dispatch_task(
+    vault_dir: Path,
+    task_path: Path,
+    *,
+    pack: str = "research-tech",
+    llm_client: Any | None = None,
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `llm_client`

(ANN401)